Restore the web component studio (#250)
omichel/webots: scripts/web_component_studio/README.md

# Web Component Studio

> The Web Component Studio is a script to create the Web components displayed in the online documentation.

Reviewer: ```suggestion
The Web Component Studio is a script used to create the 3D web components displayed in the online documentation.
```

> A Web component is a view of a Webots model, typically a robot.

Reviewer: ```suggestion
A web component is an interactive 3D view of a Webots model, typically a robot.
```

> In addition of a 3D view of the Webots model, the Web component displays the list of the robot devices.

Reviewer: ```suggestion
If the model is a robot, the web component also displays the list of devices of that robot, in addition to the 3D view of the robot.
```

Motors can be moved using sliders.
The device origin location are displayed when the mouse is over an item of the device list.

> To illustrate this, a Web component showing the Aibo ERS7 model can be shown on the top of the following page: https://cyberbotics.com/doc/guide/aibo-ers7

Reviewer: ```suggestion
An example of a web component showing the Aibo ERS7 robot is displayed on the top of the following page: https://cyberbotics.com/doc/guide/aibo-ers7
```
